core/designsystem/src/main/java/com/moya/funch/component/Button.kt

```diff
@@ -42,6 +53,12 @@ enum class FunchButtonType(val shape: Shape, val contentVerticalPadding: Dp, val
     XSmall(RoundedCornerShape(12.dp), 8.dp, funchTypography.b);
 }
 
+data class FunchIcon(
+    @DrawableRes val resId: Int,
+    val description: String,
+    val tint: Color,
+)
+
```

Comment:
```suggestion
@Immutable
data class FunchIcon(
    @DrawableRes val resId: Int,
    val description: String,
    val tint: Color,
)

```

@Immutable 어노테이션 붙여주면 좋을듯?!   


불변성을 강제하거나 최적화해서 Compose 컴파일러가 최적화될 수 있다고 알고 있습니당